query-connector/src/app/ui/designSystem/table/table.module.scss

```
@@ -1,62 +1,62 @@
 @use "../../../ui/styles/_variables" as *;
 
-.table {
-  display: flex;
-  flex-direction: column;
+.customizeTableWrapper {
+  border: solid 1px $table-border;
   border-radius: 0.25rem;
-  border: 1px solid $table-border;
-  background: $white;
-  padding: $default-table-padding 0 0 0;
   box-shadow: 0px 2px 3px 0px $drop-shadow;
+  padding: 1px;
+  background-color: $white;
+}
 
-  thead tr {
-    display: grid;
-    align-items: center;
-    border-bottom: 1px solid $table-row-border !important;
-  }
-
-  th {
-    background-color: $white !important;
-    color: $text-secondary !important;
-    text-transform: uppercase;
-    display: flex;
-    justify-content: space-between;
-    font-weight: bold;
-    word-wrap: break-word;
-    align-items: center;
-    box-sizing: border-box;
-    border: none !important;
-    padding: 0 !important;
-  }
+.customizeTable {
+  margin: 0;
 
-  tr {
-    display: grid;
-    align-items: center;
-    border-bottom: 1px solid $table-row-border;
-    margin: 0 $default-table-padding;
+  thead tr th,
+  tbody tr td {
+    padding-top: $default-table-padding;
+    padding-bottom: $default-table-padding;
+    padding-left: calc($default-table-padding - 10px);
+    padding-right: calc($default-table-padding - 10px);
```

think we can slim these down!
```suggestion
    padding: $default-table-padding calc($default-table-padding - 10px);
```

---

generally these changes are much appreciated!